examples/aishell/asr0/local/test.sh

```diff
@@ -22,7 +22,7 @@ fi
 
 if [ ${stage} -le 0 ] && [ ${stop_stage} -ge 0 ]; then
     # format the reference test file
-    python3 utils/format_rsl.py \
+    python3 ${MAIN_ROOT}/utils/format_rsl.py \
```

Comment:
why use `MAIN_ROOT` after already linked `utils` folder?

Reply:
![image](https://github.com/user-attachments/assets/555ce7af-c60c-4683-bcda-6cef1152dd34)
Using symbolic links in Git is risky because they can break easily, especially if the target paths are moved, deleted, or not compatible across different systems.

Reply:
DONE.